### ISSUE TITLE
Currently all processor jobs fail all the time

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#428]
### Context

https://github.com/AlexsLemonade/refinebio/pull/346 introduced the `processor` field for ComputationalResult objects. However it didn't populate the `Processor` table, so whenever a job finishes it tries to lookup its pipeline and ends up with an exception because it cannot look up a record that doesn't exist.

I've worked around this for the short term staging crunch for training purposes by commenting out every line where that property is set.

### Problem or idea

We should enable all of our processor jobs to finish successfully while we wait on the `Processor` table population code.

### Solution or next step

Remove the lines of code where this field is set until the lookups will succeed. When we are ready to re-enable it we should implement it in a way such that if for some reason the lookup fails we log it loudly but don't fail the job.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by kurtwheeler** [comment]
Fixed in https://github.com/AlexsLemonade/refinebio/pull/457
